exercises/mongo-shell/basic-mongo-queries.md

```diff
@@ -514,33 +721,98 @@ db.movieDetails.find({"genres":["Action", "Adventure"]},
+* Once you have the name, let's update it to reflect the correct name "Taare Zameen Par". 
 
-Some questions to consider while exploring:
-- How is this data structured?
-- Is there anything interesing to me about this data?
-- Is the data malformed, incorrect or incomplete?
-- What are the challenges you faced while exploring this data?
+Here's how to perform the update operation:
 
-**Presentation: :alarm_clock: 25 minutes**
-Each group will prepare a small, **5 minute**, 2-3 slide presentation about the datasets.
+db.movieDetails.updateOne(
+{<your uniquely identifying filter as `key: value` pair>}, {"$set": {"title": "Taare Zameen Par"}}
+)
+
+**Exercise 16** :computer: 
+
+MongoDB offers multiple operators that can be used in its update operations. In the exercise above, we looked at the `$set` operator. 
+
+|Name|Description|
+|---|---|
+|`$set`|Sets the value of a field in a document|
+|`$unset`|Removes the specified field from a document|
+|`$currentDate`|Sets the value of a field to current date, either as a Date or a Timestamp|
+|`$inc`|Increments the value of a field by a specified amount|
+|`$mul`|Multiplies the value of the field by the specified amount|
+|`$min`|Only updates the field if the specified value is less than the existing field value.|
+|`$max`|Only updates the field if the specified value is greater than the existing field value.|
+|`$setOnInsert`|Sets the value of a field if an update results in an insert of a document. Has no effect on update operations that modify existing documents.|
+|`$rename`|Renames a field|
+
+
+Take **10 minutes** :alarm_clock: to go through this list and write an interesting `$updateMany()` query. 
+
+
+:arrow_right: Note that `updateMany()` will make the same modification to all the documents that match the given filter. 
+
+---
+
+## Delete Operations
+
+The MongoDB query language supports 2 delete operations:
+* `deleteOne()`
+* `deleteMany()`
 
-Present your results from **Lab 2** and answer any questions your classmates may have.
-Be sure to include your interesting movie :clapper: queries :+1:
+**Exercise 17** :computer: 
 
+In our `movieDetails` collection, there is only 1 movie from the `year` 2018 with a `title` "QQ Speed". To perform the exercise, delete this record. 
+
+**query:**
+```javascript
+db.movieDetails.deleteOne({"title": "QQ Speed"})
+```
+Alternatively, you could use:
+```javascript
+db.movieDetails.deleteOne({"year": 2018})
+```
+since this filter would also uniquely identify the movie "QQ Speed", considering this collection has only 1 movie from that year. 
+
+**result:**
+```javascript
+{ "acknowledged" : true, "deletedCount" : 1 }
+```
+
+The result confirms that the `deleteOne()` operation was successful, at the same time telling us the number of documents that were deleted. 
+
+**Exercise 18** :computer: 
+
+Our `movieDetails` collection has a few movies from the 1800s!!! Spend **5 minutes** :alarm_clock: to delete all those movies using `deleteMany()`.
```

Comment:
```suggestion
Time to turn our attention back to the `employee_info` collection! Spend **5 minutes** :alarm_clock: to delete employees using `deleteMany()`.
```

Reply:
what about two **delete()** exercises?

- delete({department:<some department with more than one person>})
  - this would show that the "first" doc get manipulated and thus highlights unintended consequences to an operation like this.
- deleteMany({departments: "managers"})
  - this should remove all documents that belong to managers

Note: those queries are pseudo code and need to be formatted correctly.